README.md

```diff
@@ -26,53 +49,28 @@ imagelab.find_issues()
 imagelab.report()
 ```
 
-CleanVision diagnoses many types of issues, but you can also check for only specific issues:
-
+3. This step is optional. You can also specify specific issues instead of checking for all issues.
+The **Issue Key** column in the table above specifies the name for each type of issue in the CleanVision code. Use these string names to specify which types of issues should be checked.
```

Comment:
IMO this is too much details for a readme quickstart. So I'd just keep the old text:

> CleanVision diagnoses many types of issues, but you can also check for only specific issues:

I think it's fairly obvious how to check for specific types of issues based on the code itself